### PR TITLE
Data Science Team: Business establishments (land use) and jobs (100%) 

### PR DESCRIPTION
**Pull request by HarshDW09**
Hi @chamathkaaa ,

This is the final pull request

• Cleaned the dataset to ensure all necessary columns were present and handled missing values.
• Standardized and scaled the features for better model performance.
• Applied XGBoost regression to predict job density.
• Used GridSearchCV to optimize hyperparameters, resulting in improved model performance.
• Achieved high predictive accuracy with an R-squared score of 0.85.
• Identified key features impacting job density through feature importance analysis.
• Conducted spatial analysis to identify clusters of high job density.
• Provided actionable insights and recommendations for urban planners based on the model’s predictions.
• Provided suggestion to the planner.

### CODE REVIEW

**Review by chamathkaaa — APPROVED** [review]
Hi @HarshDW09 

Pull Request Review Checklist

Include .html file ✔️
Include .json file ✔️
Include .ipynb file ✔️
Does not have large number of file changes ✔️
API keys are not visible ✔️
All visualisations are labelled and legible ✔️
Contains sufficient explanations of code ✔️
Has moved .html, .json and .ipynb files to 'T2 2024' folder inside 'Ready to Publish' ✔️
Folder is deleted from Playground ✔️

Great work on your use case. You have given proper explanations, comments and the files is well organized. All files/folders are in the right place. 

Cheer,
Uvini
